index.js

```diff
@@ -1,82 +1,136 @@
-let repoList = []
+let repoList = [];
+const SEARCH_TERM = "-web-";
```

Comment:
I filtered the repos based on the string `"-v-000"` because that string was at the end of every lab in the rack I took.

I agree that it makes sense to change it to a const being that other tracks had other markers, however putting `"-web-"` as a placeholder might not be the best idea (it doesn't cover my track for example, and it would include any repo that might have the word `web` in middle of the name.

A better idea might be to move `SEARCH_TERM` into the `config.json` and let the user define it.